manifests/e/Expedition/Expedition4D/11.14.2/Expedition.Expedition4D.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Expedition.Expedition4D
+PackageVersion: 11.14.2
+PackageLocale: en-US
+Publisher: Tasman Bay Navigation Systems Ltd

```

```suggestion
Publisher: Expedition
```
This needs to match Control Panel.